backend/app/tasks/scheduler.py

```
@@ -325,50 +331,50 @@ def _check_monitoring_alerts_job(self):
             
-            print("Croatian currency rates updated successfully")
+            logger.info("Croatian currency rates updated successfully")
                 
         except Exception as e:
-            print(f"Croatian currency rates update failed: {e}")
+            logger.error(f"Croatian currency rates update failed: {e}")
     
     def _update_croatian_holidays_cache_job(self):
         """Update Croatian holidays cache."""
-        print(f"Starting Croatian holidays cache update at {datetime.now()}")
+        logger.info(f"Starting Croatian holidays cache update at {datetime.now()}")
```

Ruff (0.11.9)

Local variable `datetime` referenced before assignment

(F823)